crates/holochain/Cargo.toml

```diff
@@ -311,11 +311,10 @@ unstable-migration = [
 ]
 
 # Enables unstable warrants feature.
-unstable-warrants = [
-  "unstable-functions",
-  "holochain_state/unstable-warrants",
-  "holochain_cascade/unstable-warrants",
-]
+#
+# Invalid op warrants are now stabilized, but this feature flag is retained for
+# chain fork warrants.
+unstable-warrants = []
```

Comment:
I was wondering whether we should rename it to `unstable-fork-warrants` to make clearer that its scope has changed and normal warrants are now stable. But we can also just communicate this clearly out of band.